components/chip/API.md

```diff
@@ -22,8 +22,10 @@ import { Chip } from '@dhis2-ui/chip'
 |dragging|boolean||||
 |icon|element||||
 |marginBottom|number|`4`||`margin-bottom` value, applied in `px`|
-|marginLeft|number|`4`||`margin-left` value, applied in `px`|
-|marginRight|number|`4`||`margin-right` value, applied in `px`|
+|marginInlineStart|number|`4`||`margin-inline-start` value, applied in `px`|
```

Comment:
I thought we would want to leave the existing `marginLeft` and `marginRight` properties and not treat them as equivalent to inline-start inline-end, but that going forward it would be better to also provide for people to enter the margin in a logical-property-compliant manner (hence addition of `marginInlineStart` and `marginInlineEnd` here)